Template/NLO/SubProcesses/fks_singular.f

```diff
@@ -175,6 +206,165 @@ subroutine compute_6to5flav_cnt()
 
 
 
+      subroutine compute_alpha_cnt()
+C This is the counterterm for the change of scheme
+C in the UV renormalisation for alpha in (leptonic) PDFs
+C wrt the hard matrix element. Relevant for lepton collisions. 
+C It is called in this function such that if is included
+C in the LO cross section
```

Comment:
I do not understand the last sentences of the comment

Reply:
marco will change this call to the virtual

Reply:
done 19695c759